.github/workflows/ci.yaml

```diff
@@ -22,14 +122,26 @@ jobs:
           node-version: 16
 
       - name: Install Deps and Build
+        id: validation
         run: |
           pushd javascript
             npm ci
             npm run build
```

Comment:
i don't think we're actually running the examples here?

it's fine to spread out over multiple PRs as long as the end game here is that we have consistent behavior across all of the languages.  (I assume that's not much work.  If it is we can create tickets for the ones that don't get done in this pass.)

Reply:
We're not, which is why I renamed it from `run-samples` to `validation`, which more generally conveys that it's doing what it can on a per-language basis. I'm planning on opening an issue that enumerates the languages we still need end-to-end testing for. Hopefully that's what you're getting at?

Reply:
sure that sounds great, thanks